rosetta/retriever/retriever_test.go

```diff
@@ -202,33 +205,33 @@ func TestRetriever_Balances(t *testing.T) {
 		invoker.ScriptFunc = func(height uint64, script []byte, parameters []cadence.Value) (cadence.Value, error) {
 			assert.Equal(t, mocks.GenericBlockQualifier.Index, &height)
 			assert.Equal(t, []byte(`test`), script)
-			if assert.Len(t, parameters, 1) {
-				assert.Equal(t, cadence.NewAddress(mocks.GenericAccount.Address), parameters[0])
-			}
+			require.Len(t, parameters, 1)
+			assert.Equal(t, cadence.NewAddress(mocks.GenericAccount.Address), parameters[0])
 			return mocks.GenericAmount(0), nil
 		}
 
-		ret, err := baselineRetriever(t)
-		require.NoError(t, err)
-
-		ret.validate = validator
-		ret.generator = generator
-		ret.invoke = invoker
+		ret := retriever.New(
+			mocks.GenericParams,
+			mocks.BaselineReader(t),
+			validator,
+			generator,
+			invoker,
+			mocks.BaselineConverter(t),
+		)
 
 		blockID, amounts, err := ret.Balances(
 			mocks.GenericBlockQualifier,
 			mocks.GenericAccountID(0),
 			[]identifier.Currency{mocks.GenericCurrency},
 		)
 
-		if assert.NoError(t, err) {
-			assert.Equal(t, mocks.GenericBlockQualifier, blockID)
+		require.NoError(t, err)
+		assert.Equal(t, mocks.GenericBlockQualifier, blockID)
```

Comment:
I think now that we settled on how to do the naming, we could rename this to `mocks.GenericRosBlockID`?